src/components/Asset/AssetActions/Download/index.tsx

```diff
@@ -328,13 +329,8 @@ export default function Download({
                     <PurchaseButton isValid={isValid} />
                   </>
                 )}
-                <Field
```

Comment:
issue(blocking): we need to checkboxes here from my knowledge.
One for the portal T&C (`defautlTerms...`) and one for the asset (the one added below)